### ISSUE TITLE
REceive error in LOG (HASS 0.93.2

### ISSUE THREAD

**Issue by Klagio** [StyraHem/ShellyForHASS#24]
```

Traceback (most recent call last):
  File "/config/custom_components/shelly/switch.py", line 101, in _query_state
    auth=self._auth, timeout=5)
  File "/usr/local/lib/python3.7/site-packages/requests/api.py", line 75, in get
    return request('get', url, params=params, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/requests/api.py", line 60, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/local/lib/python3.7/site-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/requests/adapters.py", line 516, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPConnectionPool(host='192.168.1.165', port=80): Max retries exceeded with url: /relay/0 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe99fd547b8>: Failed to establish a new connection: [Errno 113] Host is unreachable'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/homeassistant/helpers/entity.py", line 220, in async_update_ha_state
    await self.async_device_update()
  File "/usr/local/lib/python3.7/site-packages/homeassistant/helpers/entity.py", line 377, in async_device_update
    await self.hass.async_add_executor_job(self.update)
  File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/config/custom_components/shelly/switch.py", line 123, in update
    self._state = self._query_state()
  File "/config/custom_components/shelly/switch.py", line 104, in _query_state
    _LOGGER.error("State query failed: " + error)
TypeError: can only concatenate str (not "ConnectionError") to str

```

**Comment by rufik** [comment]
It looks like your shelly is unavailable: `[Errno 113] Host is unreachable`.

**Comment by hakana** [comment]
Look also like a bug in error handling, I will fix that to next release.

**Comment by hakana** [comment]
Fixed in latest release, 0.0.15!
